### PR TITLE
add tolerations to values

### PR DESCRIPTION
**Pull request by srihari-tf**

### CODE REVIEW

File: charts/tfy-notebook-controller/values.yaml [hunk 1]
```diff
@@ -91,31 +91,8 @@ oauth:
 sdsServer:
   ## @param sdsServer.image image for sds-server
   image: "public.ecr.aws/truefoundrycloud/sds-server:cb6f4be9a5e342ad7a3ba4eb72bcc158501500d8"
+  ## @param sdsServer.affinity [object] Node affinity for sds-server
   affinity:
```
Comment (debajyoti-truefoundry) [inline charts/tfy-notebook-controller/values.yaml:95]: Don't you have to remove this field?

File: charts/tfy-notebook-controller/templates/oauth-secret.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{{- if .Values.oauth.enabled -}}
```
Comment (debajyoti-truefoundry) [inline charts/tfy-notebook-controller/templates/oauth-secret.yaml:1]: What happens if the `.Values.oauth` field is absent? Usually, do we need to handle that edge case?

File: charts/tfy-notebook-controller/values.yaml (hunk 1 above)
Comment (srihari-tf) [reply charts/tfy-notebook-controller/values.yaml:95]: i have removed the value but good to have since we might need it for aws

Comment (debajyoti-truefoundry) [reply charts/tfy-notebook-controller/values.yaml:95]: If the template already supports the key, why must we have it incomplete in the values file?

```suggestion
  affinity: {}
```

File: charts/tfy-notebook-controller/values.yaml [hunk 3]
```diff
@@ -145,6 +122,16 @@ sdsServer:
       ephemeral-storage: 10M
       ## @param sdsServer.resources.requests.memory The minimum memory resources requested.
       memory: 30M
+  ## @param sdsServer.tolerations [array] Spot tolerations for the notebook controller deployment.
+  tolerations:
```
Comment (debajyoti-truefoundry) [inline charts/tfy-notebook-controller/values.yaml:126]: Can we please also add the number of replicas in the values file?

**Review by debajyoti-truefoundry — APPROVED** [review]

**Pull request merged**